### PR TITLE
Initial Tests

### LINKED ISSUE #20: TESTING: Write Initial Tests
### User Story
As a dev I want to have 80% code coverage for my app.

### Context / Additional Information
Espresso tests

### Acceptance Criteria
- [ ] - 80% code coverage
- [ ]
- [ ]

### PR DESCRIPTION
**Pull request by jleeboyd**
### User Story
This PR creates the first initial tests for our app.
We will need to build out our frontend-backend connection before the db files can be tested.

### Context / Additional Information
Removed a couple unnecessary fragment files as well.

### Acceptance Criteria
- [x] - Code is 64% covered

### CODE REVIEW

**Review by Dylan-Riley — APPROVED** [review]
Thanks for getting this started, also very nice to have an example of this kind of testing for something I've worked on

File: .idea/misc.xml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project version="4">
-  <component name="ProjectRootManager" version="2" languageLevel="JDK_1_8" default="true" project-jdk-name="1.8" project-jdk-type="JavaSDK">
```
Comment (RealDogDad) [inline .idea/misc.xml:3]: Correct if wrong, but JDK_1_8 doesn't support Lambda Expressions. The ViewModel utilizes the Lambda expressions capability to push Data transaction work off of the main thread. If the JDK downgrade doesn't effect functionality, this change is conditionally approved.

**Review by RealDogDad — COMMENTED** [review]
See Comments on Lambda Expressions and the Downgrade from JDK_1_8 to JDK_1_7

File: .idea/misc.xml (hunk 1 above)
Comment (jleeboyd) [reply .idea/misc.xml:3]: Strange, I never intentionally changed the language level, good catch. I will fix and recommit

**Pull request merged**